File: Android/Java/Geofencing/app/src/main/java/com/example/geofencing/Model/WaypointsList.java
package com.example.geofencing.Model;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class WaypointsList {

    @SerializedName("hint")
    private String hint;

    @SerializedName("distance")
    private float distance;

    @SerializedName("location")
    private List location;

    @SerializedName("name")
    private String name;

    public void setHint(String hint){this.hint=hint;}
    public String getHint(){return this.hint;}

    public void setDistance(float distance){this.distance=distance;}
    public float getDistance(){return this.distance;}

    public void setLocation(List location){this.location=location;}
    public List getLocation(){return this.location;}

    public void setName(String name){this.name=name;}
    public String getName(){return this.name;}
}

File: Android/Migracion/AbuMonitor/mobile/src/main/java/com/example/comunicationwearmobile/models/maps/WaypointsList.kt
package com.example.comunicationwearmobile.models.maps

import com.google.gson.annotations.SerializedName

class WaypointsList {
    @SerializedName("hint")
    var hint: String? = null

    @SerializedName("distance")
    var distance: Float = 0f

    @SerializedName("location")
    var location: List<*>? = null

    @SerializedName("name")
    var name: String? = null
}
